images/router/haproxy/reload-haproxy

```diff
@@ -66,7 +66,7 @@ function haproxyHealthCheck() {
 }
 
 
-old_pids=$(ps -A -opid,args | grep haproxy | egrep -v -e 'grep|reload-haproxy' | awk '{print $1}' | tr '\n' ' ')
+old_pids=$(pidof haproxy)
```

Comment:
Clean! pidof is part of sysvinit-tools package and it is already available on openshift base and node images.